integration/test_collection_config.py

```diff
@@ -650,22 +650,41 @@ def test_config_export_and_recreate_from_config(collection_factory: CollectionFa
 
 def test_config_export_and_recreate_from_dict(collection_factory: CollectionFactory) -> None:
```

Comment:
we have a similar test in test_client - but we can leave it for now

Reply:
just noticed, the one in this PR tests much more so let's keep it but we should definitely merge them